courseManagementService/src/main/java/com/lms/service/service.java

```diff
@@ -0,0 +1,34 @@
+package com.lms.service;
+
+import com.lms.repo.Leo_Repo;
+import com.lms.ecxeption.ResourceNotFoundException;
+import com.lms.model.lesson;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+
+public class service {
+
+    Leo_Repo repo;
```

Comment:
follow the standers in variable naming

Reply:
ok, Are there any problems in the code?? 
you can any advice, if the code not good

Reply:
Great Job